tasks/sign-binaries/sign-binaries.yaml

```diff
@@ -0,0 +1,150 @@
+      type: string
+      description: Secret to interact with Quay
+    - name: windowsCredentials
+      type: string
+      description: Secret to interact with the Windows signing host
+    - name: pipelineRunUid
+      type: string
+      description: Unique identifier for the pipeline run
+  volumes:
+    - name: windows-ssh-key-vol
+      secret:
+        secretName: windows-ssh-key
+  workspaces:
+    - name: data
+      description: Workspace to save the results to
+  results:
+    - name: unsignedWindowsDigest
+      type: string
+      description: |
+        The digest of the unsigned content pushed using ORAS for signing hosts
+    - name: signedWindowsDigest
+      type: string
+      description: |
+        The digest of the unsigned content pushed using ORAS for signing hosts
+  steps:
+    - name: push-unsigned-using-oras
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      script: |
+        #!/bin/bash
+
+        # TODO CLDX-134
+
+        output=$(oras push "$(params.quayURL)/unsigned" .)
+        windows_digest=$(echo "$output" | grep 'Digest:' | awk '{print $2}')
+        echo "Digest for windows content: $windows_digest"
+        echo -n "$windows_digest" > "$(results.unsignedWindowsDigest.path)"
+
+    # - name: sign-mac-binaries
+    #   image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+    #   script: |
+    #     #!/bin/bash
+    #     # TODO CLDX-79
+    - name: sign-windows-binaries
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      volumeMounts:
+        - name: windows-ssh-key-vol
+          mountPath: "/etc/secrets"
+          readOnly: true
+      env:
+        - name: WINDOWS_USER
+          valueFrom:
+            secretKeyRef:
+              name: $(params.windowsCredentials)
+              key: username
+        - name: WINDOWS_PORT
+          valueFrom:
+            secretKeyRef:
+              name: $(params.windowsCredentials)
+              key: port
+        - name: WINDOWS_HOST
+          valueFrom:
+            secretKeyRef:
+              name: $(params.windowsCredentials)
+              key: host
+        - name: QUAY_USER
+          valueFrom:
+            secretKeyRef:
+              name: $(params.quaySecret)
+              key: username
+        - name: QUAY_PASS
+          valueFrom:
+            secretKeyRef:
+              name: $(params.quaySecret)
+              key: password
+      script: |
+        #!/bin/bash
+        mkdir -p ~/.ssh
+        cp "/etc/secrets/id_rsa" ~/.ssh/id_rsa
+        chmod 600 ~/.ssh/id_rsa
+        unsigned_digest=$(cat "$(results.unsignedWindowsDigest.path)")
+        # EOF should be unquoted here to allow for variable expansion on the client side
+        # shellcheck disable=SC2086,SC2087
+        ssh -i ~/.ssh/id_rsa -o StrictHostKeyChecking=no -p "${WINDOWS_PORT}" "${WINDOWS_USER}@${WINDOWS_HOST}" << EOF
```

Comment:
Maybe it's ok for now, but it would be great if we could have a fingerprint of the windows host and inject it into `~/.ssh/known_hosts` and then the host checking could happen as normal.